meteor/lib/api/pubsub.ts

```diff
@@ -162,6 +165,10 @@ export interface PubSubTypes {
 	[PubSub.studios]: (selector: MongoQuery<DBStudio>, token?: string) => DBStudio
 	[PubSub.studioOfDevice]: (deviceId: PeripheralDeviceId, token?: string) => DBStudio
 	[PubSub.timeline]: (selector: MongoQuery<TimelineComplete>, token?: string) => TimelineComplete
+	[PubSub.timelineDatastore]: (
+		selector: MongoQuery<DBTimelineDatastoreEntry>,
```

Comment:
I would like to see this publication be not using a mongo query, and instead be using a studioId, or whatever identifier is relevant